diff --git a/node_modules/@types/react-sortable-tree/index.d.ts b/node_modules/@types/react-sortable-tree/index.d.ts
index 57f018f..56a5e5d 100644
--- a/node_modules/@types/react-sortable-tree/index.d.ts
+++ b/node_modules/@types/react-sortable-tree/index.d.ts
@@ -19,6 +19,22 @@ export type TreeItem<T = {}> = T & {
     title?: React.ReactNode | undefined;
     subtitle?: React.ReactNode | undefined;
     expanded?: boolean | undefined;
+    metadata: { 
+        available: boolean, 
+        deviceId: string, 
+        value: number, 
+        type: string, 
+        // modal
+        customName?: string;
+        icon?: string;
+        parentNodeCustomName?: string;
+        active?: boolean;
+        origin?: string;
+        devCustomName?: string;
+        destination?: string;
+        classification?: string;
+        phase?: string;
+    };
     children?: Array<TreeItem<T>> | GetTreeItemChildrenFn<T> | undefined;
 };
 
