diff --git a/node_modules/@types/react-sortable-tree/index.d.ts b/node_modules/@types/react-sortable-tree/index.d.ts
index 57f018f..65fc1f2 100644
--- a/node_modules/@types/react-sortable-tree/index.d.ts
+++ b/node_modules/@types/react-sortable-tree/index.d.ts
@@ -19,6 +19,46 @@ export type TreeItem<T = {}> = T & {
     title?: React.ReactNode | undefined;
     subtitle?: React.ReactNode | undefined;
     expanded?: boolean | undefined;
+    metadata: { 
+        available: boolean, 
+        deviceId: string, 
+        value: number, 
+        type: string, 
+        roomName?: string,
+        idUser?: string;
+        // modal
+        customName?: string;
+        icon?: string;
+        parentNodeCustomName?: string;
+        active?: boolean;
+        origin?: string;
+        devCustomName?: string;
+        destination?: string;
+        classification?: string;
+        phase?: string;
+        charts?: {
+            realtime?: {
+                currentIntensity?: boolean;
+                voltage?: boolean;
+                power?: boolean;
+            },
+            history?: {
+                currentIntensity?: boolean;
+                voltage?: boolean;
+                power?: boolean;
+                energy?: boolean;
+            },
+            profiles?: {
+                spring?: boolean;
+                summer?: boolean;
+                autumn?: boolean;
+                winter?: boolean;
+                winterVsSummer?: boolean;
+                electricDemand?: boolean;    
+            },
+        }
+
+    };
     children?: Array<TreeItem<T>> | GetTreeItemChildrenFn<T> | undefined;
 };
 
